packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -116,36 +121,37 @@ class ChromeLauncher {
           return null;
         });
     final Uri remoteDebuggerUri = await _getRemoteDebuggerUrl(Uri.parse('http://localhost:$port'));
-
     return _connect(Chrome._(
       port,
       ChromeConnection('localhost', port),
       process: process,
-      dataDir: dataDir,
       remoteDebuggerUri: remoteDebuggerUri,
-    ));
+    ), skipCheck);
   }
 
-  static Future<Chrome> _connect(Chrome chrome) async {
+  static Future<Chrome> _connect(Chrome chrome, bool skipCheck) async {
     if (_currentCompleter.isCompleted) {
       throwToolExit('Only one instance of chrome can be started.');
     }
     // The connection is lazy. Try a simple call to make sure the provided
     // connection is valid.
-    try {
-      await chrome.chromeConnection.getTabs();
-    } catch (e) {
-      await chrome.close();
-      throwToolExit(
-          'Unable to connect to Chrome debug port: ${chrome.debugPort}\n $e');
+    if (!skipCheck) {
+      try {
+        await chrome.chromeConnection.getTabs();
+      } catch (e) {
+        await chrome.close();
+        print('here');
```

Comment:
stray debug print

Reply:
Fixed in https://github.com/flutter/flutter/pull/39751